content/en/docs/concepts/secure-deployment.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Securely deploying Notation"
+description: "Best practices for securely deploying Notation"
+type: docs
+weight: 2
+---
+
+A deployment of Notation, specifically the `notation` CLI, is used for signing and verifying container images. The `notation` CLI is usually deployed on a development computer or in a CI/CD pipeline, such as GitHub Actions.
+
+A deployment has several sensitive components:
+
+- [Trust policies]({{< ref "/docs/how-to/manage-trust-policy" >}})
+- [Trust stores and certificates]({{< ref "/docs/concepts/directory-structure" >}})
+- Plugins
```

Comment:
will this link to [this page](https://github.com/notaryproject/notaryproject.dev/pull/219)?

Reply:
@byronchien yes, as soon as the PR is merged. I'll provide the update in a follow up PR.